src/libraries/Microsoft.Win32.SystemEvents/ref/Microsoft.Win32.SystemEvents.cs

```diff
@@ -4,6 +4,8 @@
 // Changes to this file must follow the https://aka.ms/api-review process.
 // ------------------------------------------------------------------------------
 
+using System;
+
```

Comment:
Please undo this change. Reference source files don't use usings.

Reply:
You may want to regenerate the reference source using https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/updating-ref-source.md (`/t:GenerateReferenceAssemblySource`)